compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BIntersectionType.java

```diff
@@ -94,6 +95,11 @@ public boolean isNullable() {
 
     @Override
     public String toString() {
+        Name name = this.tsymbol.name;
+        if (name != null && !name.value.isEmpty() && !name.value.startsWith("$")) {
```

Comment:
Will checking the ANONYMOUS flag work instead of `name.value.startsWith("$")`?

Reply:
Let me check, will add the fixing issues

Reply:
It works thank you.